test/topology_experimental_raft/test_tablets.py

```diff
@@ -1430,3 +1435,31 @@ async def assert_empty_table():
 
     logger.info("Verify data is not resurrected")
     await assert_empty_table()
+
+@pytest.mark.repair
+@pytest.mark.asyncio
+async def test_tablet_manual_repair(manager: ManagerClient):
+    servers, cql, hosts, table_id = await create_table_insert_data_for_repair(manager, fast_stats_refresh=False)
+    token = -1
```

Comment:
This assumes power-of-two tablet boundaries. Let's not do that. Later when we lift that, we'll have a lot of work.

Reply:
It repairs the tablet that owns the token -1. It does not require the boundary to be -1.  So without power-of-two tablet boundaries, the test will continue to work.

Reply:
But get_tablet_task_id() compares the token with "last_token" exactly.

Reply:
Ah, it's not used by the test. But then we should drop get_tablet_task_id(), or adjust it.

Reply:
It is used for another test that we will add soon. I will drop the helper for now.